### PR TITLE
Adding in a nice fade in for the user tiles.

### PR DESCRIPTION
**Pull request by julie-dhs-isobar**

### CODE REVIEW

File: src/app/components/index.js [hunk 1]
```diff
@@ -79,8 +79,20 @@ var ResultView = Marionette.ItemView.extend({
         "request": "onRequest"
     },
     onRender: function() {
-        this.$el.html(this.model.attributes.results);
-        this.model.clear({ silent: true });
```
Comment (lee-dhs-isobar) [inline src/app/components/index.js]: reason you removed this?

Comment (julie-dhs-isobar) [inline src/app/components/index.js]: I moved it to line 91-92. That's how I'm appending each card after the image has loaded.

Comment (lee-dhs-isobar) [inline src/app/components/index.js]: Aah ok, `cardContainer.model.clear({ silent: true });` but this is then run multiple times in the `$.each`, it needs to run once

Comment (julie-dhs-isobar) [inline src/app/components/index.js]: Gotcha. Updated to move it outside the each.

**Pull request merged**